frontend/src/components/common/footer/index.css

```diff
@@ -1,98 +1,98 @@
-.footer {
-  margin-top: auto;
-  bottom: 0;
-  height: 100px;
-  background-color: #519b46;
-  position: relative;
-}
-
-.footer_link {
-  display: inline-block !important;
-}
-
-.menu-text {
-  /* color: white; */
-}
-
-.foot_text {
-  color: white;
-  text-decoration: none;
-}
-li {
-  color: white !important;
-  text-decoration: none !important;
-}
-
-nav {
-  text-decoration: none;
-}
-
-.topFooter {
-  background-color: blue;
-}
-
-.box {
-  height: 60px;
-  border: 2px solid red;
-  margin: 4%;
-}
-
-.flex-container {
-  display: flex;
-  justify-content: center;
-}
-
-.pic-container {
-  background: pink;
-  min-width: 200px;
-  height: 150px;
-  border-radius: 30px;
-  margin: 20px;
-  overflow: hidden;
-  transform: skew(20deg);
-  display: flex;
-  align-items: center;
-}
-
-.pic {
-  width: 135%;
-  position: absolute;
-  left: -35px;
-  transform: skewX(-20deg);
-  object-fit: cover;
-}
-
-.foot_head {
-  display: inline;
-  margin: 20px;
-}
-
-.footerImageHeader {
-  box-sizing: border-box;
-  text-align: center !important;
-}
-
-.h2-heading {
-  display: inline-block;
-  padding-right: 30px !important;
-  text-align: center !important;
-}
-
-.h2-heading h2 {
-  color: black !important;
-  display: inline-block !important;
-  font-size: 36pt;
-  font-family: "Gotham Medium", "Arial Narrow", Arial, sans-serif;
-  text-align: center !important;
-}
-/* Moves the inspiriing */
-.movefont {
-  margin-right: 95px !important;
-}
-
-.moveConnecting {
-  margin-right: 35px !important;
-}
-#tsocial {
-  display: inline-block;
-}
+.footer {
+  margin-top: auto;
+  bottom: 0;
+  height: 100px;
+  background-color: #519b46;
+  position: relative;
+}
+
+.footer_link {
+  display: inline-block !important;
+}
+
+.menu-text {
+  /* color: white; */
+}
+
+.foot_text {
+  color: white;
+  text-decoration: none;
+}
+li {
```

Comment:
This needs to be deleted and moved to a class or ID; we shouldn't apply styles to basic HTML elements.